### PR TITLE
feat: wider proposal view

### LINKED ISSUE #1278: Wide(r) single proposal view
Part of https://github.com/decentraland/governance/issues/1263

https://www.figma.com/file/e0IFJjplUdYdQJ2NtLwvMf/DAO-v2-(Guane)?type=design&node-id=4759-681&mode=design

Image

- Columns on both sides have fixed width
- Center (content) column is fluid
- Optional: we may want to increase font-size + line-height on content (screenshot is using bigger text)

### PR DESCRIPTION
**Pull request by 1emu**
Closes #1278 

- Responsive 2 column design
- Fixed issue with long links or addresses on mobile view

image

image

image

image

image

### CODE REVIEW

File: src/pages/proposal.css [hunk 1]
```diff
@@ -1,15 +1,39 @@
-@media (max-width: 767px) {
-  .SignIn.center .ui.primary.button {
-    display: none;
-  }
+.ProposalDetailPage {
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 1410px;
```
Comment (ncomerci) [inline src/pages/proposal.css:4]: if you use the component WiderContainer, you shouldn't have to handle this

File: src/pages/proposal.tsx [hunk 2]
```diff
@@ -381,86 +381,79 @@ export default function ProposalPage() {
...
-                votes={votes}
-                votesLoading={isLoadingVotes}
-                isCoauthor={isCoauthor}
-                isOwner={isOwner}
-              />
-            </Grid.Column>
-          </Grid.Row>
-        </Grid>
-      </Container>
+      <div className={'ProposalDetailPage'}>
```
Comment (ncomerci) [inline src/pages/proposal.tsx:384]: related to previous comment

File: src/components/Layout/Navigation.css
Comment (ncomerci) [inline src/components/Layout/Navigation.css:1]: why have you split `.Navigation .dcl.tabs .ui.container` into two parts? 🤔

File: src/components/FloatingBar/FloatingBar.css [hunk 3]
```diff
@@ -8,12 +8,12 @@
   border: 1px solid var(--black-300);
   background: var(--white-900);
   box-shadow: 0 0 25px 0 var(--alpha-black-400);
-  padding: 12px 18px 12px 16px;
+  padding: 12px 1em 12px 1em;
```
Comment (andyesp) [inline src/components/FloatingBar/FloatingBar.css:11]: You could write this like `padding: 12px 1em` (or 1em 12px, don't remember)

File: src/pages/proposal.css [hunk 4]
```diff
@@ -1,15 +1,34 @@
...
 }
 
-.ProposalDetailPage .ui.grid > .row > .column.ProposalDetailPage__Description {
-  width: 72% !important;
+.ProposalDetailPage__Description {
+  max-width: 100%;
+  padding-right: 0;
+}
+
+@media (min-width: 769px) {
```
Comment (andyesp) [inline src/pages/proposal.css:12]: This should be 768

File: src/pages/proposal.css [hunk 5]
```diff
@@ -30,21 +49,16 @@
...
-@media only screen and (min-width: 320px) and (max-width: 767px) {
-  .ProposalDetailPage .ui.grid > .row > .column.ProposalDetailPage__Description {
-    width: 100% !important;
-    padding: 1rem 0 !important;
-  }
-
-  .ProposalDetailPage .ui.grid > .row > .column.ProposalDetailActions {
-    width: 100% !important;
-    padding: 1rem 0 !important;
+@media only screen and (min-width: 320px) and (max-width: 768px) {
```
Comment (andyesp) [inline src/pages/proposal.css:60]: We could remove min-width: 320px as we don't care about what happens below 320.

**Review by andyesp — APPROVED** [review]

File: src/components/FloatingBar/FloatingBar.css (hunk 3 above)
Comment (1emu) [reply src/components/FloatingBar/FloatingBar.css:11]: padding: 12px 1em

**Pull request merged**